proposals/dao/swap_dpi_to_dai.ts

```diff
@@ -0,0 +1,172 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { TransactionResponse } from '@ethersproject/providers';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner, overwriteChainlinkAggregator, time } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+const toBN = ethers.BigNumber.from;
+
+/*
+Swap DPI for DAI using an LBP Swapper
+*/
+
+const fipNumber = 'swap_dpi_to_dai';
+
+// LBP Swapper config
+const LBP_FREQUENCY = 86400 * 30; // 1 month in seconds
```

Comment:
1 month is probably too long, I'd lean closer to 2 weeks

Reply:
Reduced to 2 weeks